ecommerce-ui/src/Apartment.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+// import { Container, Row, Col } from 'reactstrap';
+
+class Apartment extends Component {
+    render () {
+        const apartment = this.props.post
```

Comment:
Remember to continue validating props with Proptypes in your work. It is commonplace in larger projects.